lib/sentry/logger_backend.ex

```diff
@@ -141,7 +148,10 @@ defmodule Sentry.LoggerBackend do
 
     sentry_context =
       if sentry_context_from_meta || sentry_context_from_sentry do
-        Map.merge(sentry_context_from_meta || %{}, sentry_context_from_sentry || %{})
+        Map.merge(
+          Map.new(sentry_context_from_meta || %{}),
+          Map.new(sentry_context_from_sentry || %{})
+        )
```

Comment:
Unnecessary change?

Reply:
Not that it matters given your request to revert changes to the backend, but without this,
`Logger.error("Testing error", sentry: [tags: %{explicit: "tag"}])` code fails even without other changes, as `sentry_context_from_meta` is a list, not a map, so `Map.merge` fails - it's just not covered by existing tests it seems

Reply:
Aaaah I see sorry, my eyes mixed and I thought this was just a formatting change, I now see that `Map.new/1` is called on both lines. But this is still a fix even if you'll revert the rest of the changes here, won't it? Maybe it would be nice to revert it here and open in as a separate PR with a related regression test case.

Reply:
Reverted